### ISSUE TITLE
Can't convert lead to deal

### ISSUE THREAD

**Issue by metalmon** [frappe/crm#429]
Post request returns code 417
![image](https://github.com/user-attachments/assets/c097560e-3ada-4046-be8e-139556e62750)

**Comment by metalmon** [comment]
Local development setup in docker
Frappe version-15
ERPNext version-15
Frappe CRM
Frappe HelpDesk
Frappe Drive

**Comment by metalmon** [comment]
Yes, the same error. If they don't fix it for a long time, then I'll do it myself, send a PR

**Comment by shariquerik** [comment]
@metalmon Give me an reproducible example so that I can fix it. It will help a lot if you can send some screenshots or recording

**Comment by metalmon** [comment]
@shariquerik Change the names of the lead or deal statuses in the admin panel, preferably not in Latin. Perhaps the fact is that the set of stages of the funnel are hardcoded

**Comment by shariquerik** [comment]
No Statuses are not hardcoded. It does work with any Statuses. Closing.
Reopen if you have an example of reproducible issue with screenshots and screen recordings

**Comment by anushmohan95** [comment]
@shariquerik 

Thank you for looking into this.
Please see below video.

https://github.com/user-attachments/assets/872b4672-6c70-47cc-8752-c2e83f9913b4
